eva/server/networking_utils.py

```diff
@@ -79,3 +81,10 @@ def set_socket_io_timeouts(transport, seconds, useconds=0):
         return True
     except OSError:
         return False
+
+
+def serialize_message(message: str):
```

Comment:
Shall we use `typing.Any` for the type hint of `message`?

Reply:
Done